components/atoms/StatusBadge.js

```diff
@@ -6,9 +6,18 @@ import propTypes from 'prop-types'
 
 export default function StatusBadge(props) {
   return (
-    <div className="flex  justify-end rounded-t-lg ">
+    <div className="flex justify-end rounded-t-lg ">
       <h2
-        className={`font-bold font-display text-white text-lg px-5 py-1 sm:mb-5 w-full sm:w-1/3 sm:mr-14 rounded-t-lg sm:rounded-t-none sm:rounded-b-lg ${props.className}`}
+        className={`font-medium font-display text-black text-lg px-5 py-1 sm:mb-5 w-full sm:w-1/3 sm:mr-14 rounded-t-lg sm:rounded-t-none sm:rounded-b-lg 
+        ${
+          props.status === 'In payment'
+            ? 'bg-status-inPayment'
+            : props.status === 'Benefit update'
+            ? 'bg-status-benefitUpdate'
+            : props.status === 'Application received'
+            ? 'bg-status-applicationReceived'
+            : 'bg-status-inactive'
+        }`}
```

Comment:
This won't work if you translate to french. 
One option to fix it would be to pass the code to the component. Then this check can be made using the code (importing the code const). And you would then load the translation from the translation file based on the code.